app/src/components/table/CellWithControlsWrapper.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { PropsWithChildren, ReactNode } from "react";
+import { css } from "@emotion/react";
+
+const cellWithControlsWrapCSS = css`
+  position: relative;
+  min-height: 75px;
+  .controls {
+    transition: opacity 0.2s ease-in-out;
+    opacity: 0;
+    display: none;
+    z-index: 1;
+  }
+  &:hover .controls {
+    opacity: 1;
+    display: flex;
+    // make them stand out
+    .ac-button {
+      border-color: var(--ac-global-color-primary);
+    }
+  }
+`;
+
+const cellControlsCSS = css`
+  position: absolute;
+  top: -23px;
+  right: 0px;
+  display: flex;
+  flex-direction: row;
+  gap: var(--ac-global-dimension-static-size-100);
+`;
+
+/**
+ * Wraps a cell to provides space for controls that are shown on hover.
+ */
+export function CellWithControlsWrap(
```

Comment:
this is generalized (name component already was) and pulled from experiments compare table